client-web/src/exlibris-app/exlibris-book-grid.html

```diff
@@ -0,0 +1,107 @@
+<!--
+  ~ Copyright 2018, TeamDev Ltd. All rights reserved.
+  ~
+  ~ Redistribution and use in source and/or binary forms, with or without
+  ~ modification, must retain the above copyright notice and the following
+  ~ disclaimer.
+  ~
+  ~ THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS AND CONTRIBUTORS
+  ~ "AS IS" AND ANY EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT
+  ~ LIMITED TO, THE IMPLIED WARRANTIES OF MERCHANTABILITY AND FITNESS FOR
+  ~ A PARTICULAR PURPOSE ARE DISCLAIMED. IN NO EVENT SHALL THE COPYRIGHT
+  ~ OWNER OR CONTRIBUTORS BE LIABLE FOR ANY DIRECT, INDIRECT, INCIDENTAL,
+  ~ SPECIAL, EXEMPLARY, OR CONSEQUENTIAL DAMAGES (INCLUDING, BUT NOT
+  ~ LIMITED TO, PROCUREMENT OF SUBSTITUTE GOODS OR SERVICES; LOSS OF USE,
+  ~ DATA, OR PROFITS; OR BUSINESS INTERRUPTION) HOWEVER CAUSED AND ON ANY
+  ~ THEORY OF LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY, OR TORT
+  ~ (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE
+  ~ OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
+  -->
+
+<link rel="import" href="../../bower_components/polymer/polymer-element.html">
+<link rel="import" href="../../bower_components/polymer/lib/elements/dom-if.html">
+<link rel="import" href="../../bower_components/iron-list/iron-list.html">
+<link rel="import" href="../../bower_components/paper-styles/typography.html">
+
+<link rel="import" href="../exlibris-app/exlibris-book-tile.html">
+<link rel="import" href="./book-values.html">
+
+<dom-module id="exlibris-book-grid">
+    <template>
+        <style>
+            :host {
+                display: block;
+            }
+
+            .center {
+                @apply --paper-font-subhead;
+                margin-top: 250px;
+                text-align: center;
+                color: var(--paper-grey-700);
+            }
+
+            .hidden {
+                display: none;
+            }
+        </style>
+        <div class$="[[_isBooksEmpty(books)]]">[[message]]</div>
+
+        <iron-list items="[[books]]" as="book" scroll-target="document" grid>
+            <template>
+                <div>
+                    <exlibris-book-tile book="[[book]]"></exlibris-book-tile>
+                </div>
```

Comment:
@YuriiHaidamaka Why do you think div fixes layou?

Reply:
@dkalinin I think it is kind of magic).  I used a demo example of `<iron-list grid>`.
` "the list template item must have both fixed width and height"`.  If I delete `<div>` element, the width and height are still in the `:host` of `<exlibris-book-tile>` but it does not help. How to solve this problem ?

Reply:
@YuriiHaidamaka 
Ok, hint:
Html tags have default `display` value: https://www.impressivewebs.com/default-css-display-values-html-elements/

Reply:
@dkalinin  It's certainly very cool, but it does not help me at all. In all examples that I have found there is this `div`. I inspected this page in browser and found that this div has `style` attribute  of the next type : `style="transform: translate3d(98px, 0px, 0px);`and for each tile this attribute is different.  It makes me think that `div` is there is needed for something.